docs/docs-beta/docs/guides/build/project-structure/components.md

```diff
@@ -37,87 +47,180 @@ Commands:
   deployment      Commands for operating on deployment directories.
 
 Options:
-  --builtin-component-lib TEXT  Specify a builitin component library to use.
-  --verbose                     Enable verbose output for debugging.
-  --disable-cache               Disable caching of component registry data.
-  --clear-cache                 Clear the cache before running the command.
+  --clear-cache                 Clear the cache.
   --rebuild-component-registry  Recompute and cache the set of available component types for the current environment.
                                 Note that this also happens automatically whenever the cache is detected to be stale.
-  --cache-dir PATH              Specify a directory to use for the cache.
   -v, --version                 Show the version and exit.
   -h, --help                    Show this message and exit.
+
+Global options:
+  --use-dg-managed-environment / --no-use-dg-managed-environment
+                                  Enable management of the virtual environment with uv.
+  --builtin-component-lib TEXT    Specify a builitin component library to use.
+  --verbose                       Enable verbose output for debugging.
+  --disable-cache                 Disable the cache..
+  --cache-dir PATH                Specify a directory to use for the cache.
 ```
 
-We're going to generate a new code location.
+We want to scaffold a new code location:
 
 ```bash
-dg code-location generate jaffle_platform
+$ dg code-location scaffold jaffle-platform
+
+Creating a Dagster code location at .../jaffle-platform.
+Scaffolded files for Dagster project in .../jaffle-platform.
+Using CPython 3.12.4
+Creating virtual environment at: .venv
+Resolved 73 packages in 398ms
+   Built jaffle-platform @ file:///.../jaffle-platform
+...
 ```
 
-Let's have a look at what it generated:
+This built a code location at `jaffle-platform` and initialized a new Python
+virtual environment inside of it. When using `dg`'s default environment
+management behavior, you won't need to worry about activating this virtual environment yourself.
+
+Let's have a look at the scaffolded files:
 
 ```bash
-cd jaffle_platform && tree
+$ cd jaffle-platform && tree
+
+.
+├── jaffle_platform
+│   ├── __init__.py
+│   ├── __pycache__
```

Comment:
do the pycache files show up after the first invocation?

even if they do, maybe we should just hide them from this view just to simplify things